tracing/src/lib.rs

```diff
@@ -1137,6 +1137,85 @@ pub mod __macro_support {
                 .finish()
         }
     }
+
+    /// Implementation of [autoref specialization] for span/event value capture.
+    ///
+    /// Used by the `__tracing_capture_value!` macro by importing all of the
+    /// traits in this module, which all provide `__tracing_capture_value()`,
+    /// and then calling `(&&value).__tracing_capture_value()`.
+    ///
+    /// Method resolution checks `&&T`, then `&T`, and then `T` for the
```

Comment:
I just realized I wrote this backwards

```suggestion
    /// Method resolution checks `T`, then `&T`, and then `&&T` for the
```